homeassistant/components/octoprint/__init__.py

```diff
@@ -154,7 +155,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if DOMAIN not in hass.data:
         hass.data[DOMAIN] = {}
 
-    websession = async_get_clientsession(hass)
+    verify_ssl = entry.data.get(CONF_VERIFY_SSL, True)
```

Comment:
We should update the existing configuration entries that lack `CONF_VERIFY_SSL` (as in a little migration).